File: OptiPlanBackend/OptiPlanBackend/Models/Project.cs
namespace OptiPlanBackend.Models
{
    public class Project
    {
        public Guid Id { get; set; }
        public string Title { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }

        public Guid OwnerId { get; set; }
        public User Owner { get; set; }

        public Team Team { get; set; }

        public ICollection<WorkItem> Tasks { get; set; } = new List<WorkItem>();

     
        public ICollection<Sprint> Sprints { get; set; } = new List<Sprint>();
    }
}

File: optiplan-frontend/src/app/models/project.ts
import { ProjectParticipation } from "./project-participation";
import { Team } from "./team";
import { User } from "./user";


export interface Project {
  id: string;
  title: string;
  description: string;
  startDate: string;
  endDate: string;
  ownerId: string;
  owner: User;
  team: Team;
  participants: ProjectParticipation[];
}
